agent/main.go

```diff
@@ -71,6 +72,23 @@ func (l *labelFlags) Set(value string) error {
 	}
 }
 
+func setFlagIfNotSet(flagName string) error {
+	found := false
+	flag.Visit(func(f *flag.Flag) {
+		if f.Name == flagName {
+			found = true
+		}
+	})
+
+	// checking if log level flag has been explicitly set
+	// else setting the flag at 1
+	if !found {
+		err := flag.Set("v", "1")
+		return err
+	}
+	return nil
+}
+
```

Comment:
Question: Doesn't `klog.Initflags` handle this. Also I guess default verbosity level should be 0 to capture all Info, error logs. And for installer, we could use `log.V(1).Info`. @anusha94 @pshail wdyt?

Reply:
`klog.Initflags()` enables `v` flag and a set of other flags. The above code-block sets the default `v` level at 1 since all our Info logs will be generated at 1. However, this won't be required if we set the default verbosity level 0 and installer log verbosity at 1

Reply:
@anusha94 @pshail let me know how we want to move forward with this.